DataProcessing/ExtractAlphaTrueBeatsConverter.cs

```diff
@@ -349,20 +370,12 @@ protected void WriteToFile(Dictionary<string, List<ExtractAlphaTrueBeat>> proces
                         .ToList();
                 }
 
+                outputData.AddRange(trueBeatData);
                 var outputDataLines = outputData
+                    .OrderBy(x => x.Time)
                     .Select(ToCsv)
+                    .Distinct()
```

Comment:
Note there is no guarantee of order here see https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.distinct?view=net-5.0#remarks